config/src/test/groovy/org/springframework/security/config/annotation/web/configurers/ExpressionUrlAuthorizationsTests.groovy

```diff
@@ -451,6 +452,94 @@ public class ExpressionUrlAuthorizationConfigurerTests extends BaseSpringSpec {
 		}
 	}
 
+	@EnableWebSecurity
```

Comment:
Would you mind placing these new tests in a Java class, like `ExpressionUrlAuthorizationConfigurerAnyMatcherTests`? We're in the process of migrating all Groovy tests to Java.

Reply:
> Would you mind placing these new tests in a Java class, like `ExpressionUrlAuthorizationConfigurerAnyMatcherTests`? We're in the process of migrating all Groovy tests to Java.

Fixed